sites/next.skeleton.dev/src/content/docs/components/tab/react.mdx

```diff
@@ -0,0 +1,88 @@
+---
+layout: '@layouts/LayoutDoc.astro'
+---
+
+export const components = componentSet;
+
+import { Page as Example } from '@examples/components/tabs/Example.tsx';
+import ExampleRaw from '@examples/components/tabs/Example.tsx?raw';
+import { Page as IconsExample } from '@examples/components/tabs/IconsExample.tsx';
+import IconsExampleRaw from '@examples/components/tabs/IconsExample.tsx?raw';
+import { Page as VerticalIconsExample } from '@examples/components/tabs/VerticalIconsExample.tsx';
+import VerticalIconsExampleRaw from '@examples/components/tabs/VerticalIconsExample.tsx?raw';
+import { Page as StretchExample } from '@examples/components/tabs/StretchExample.tsx';
+import StretchExampleRaw from '@examples/components/tabs/StretchExample.tsx?raw';
+import { Page as JustifyExample } from '@examples/components/tabs/JustifyExample.tsx';
+import JustifyExampleRaw from '@examples/components/tabs/JustifyExample.tsx?raw';
+import { Page as FocusableContentExample } from '@examples/components/tabs/FocusableContentExample.tsx';
+import FocusableContentExampleRaw from '@examples/components/tabs/FocusableContentExample.tsx?raw';
+
+{/* --- */}
+
+<Preview client:load>
+	<Fragment slot="preview">
+		<Example client:load />
+	</Fragment>
+	<Fragment slot="code">
+		<Code code={ExampleRaw} lang="tsx" />
+	</Fragment>
+</Preview>
+
+## Layout
+
+Easily customize tabs content with icons and arrange them in any combination you prefer.
+
+<Preview client:load>
+	<Fragment slot="preview">
+		<IconsExample client:load />
+	</Fragment>
+	<Fragment slot="code">
+		<Code code={IconsExampleRaw} lang="tsx" />
+	</Fragment>
+</Preview>
+
+<Preview client:load>
+	<Fragment slot="preview">
+		<VerticalIconsExample client:load />
+	</Fragment>
+	<Fragment slot="code">
+		<Code code={VerticalIconsExampleRaw} lang="tsx" />
+	</Fragment>
+</Preview>
+
+## Stretch
+
+Tabs can be stretched to fill the available space.
```

Comment:
Append at the end: "...using the `x` prop." (whichever prop is relevant here)

Reply:
@endigo9740 we are using classes to control the width of the tab control, so I am not sure what to write here...

Reply:
@Mahmoud-zino seem like we would want to implement a `width` prop, right? This illustrates a use case for it.